deco-sites/otica-isabela
# Feature/custom size filters

## What is this contribution about?

Created custom size filters for Search & PLP pages.

## How to test it?

Go to any Search or Category pages and check for the sizes filters, custom must appear if available.
